build/index.ts

```diff
@@ -631,6 +631,9 @@ export async function buildDocument(
 
   injectSource(doc, document, metadata);
 
+  if (document.metadata["short-title"]) {
+    doc.short_title = document.metadata["short-title"];
```

Comment:
Let's use the same casing as `pageTitle`:

```suggestion
    doc.shortTitle = document.metadata["short-title"];
```